services/drone/drone-build.service.js

```diff
@@ -85,11 +88,7 @@ module.exports = class DroneBuild extends BaseJsonService {
       qs: {
         ref: branch ? `refs/heads/${branch}` : undefined,
       },
-    }
-    if (serverSecrets.drone_token) {
-      options.headers = {
-        Authorization: `Bearer ${serverSecrets.drone_token}`,
-      }
+      headers: this.authHelper.bearerAuthHeader,
```

Comment:
If there's no `drone_token` set in the config, then this ends up as `headers: undefined` which is why we no longer need the conditional check that was done before, correct?

Reply:
Correct!